docs/settings/index.rst

```diff
@@ -479,6 +479,36 @@ UPLOAD_SESSION_EXPIRED_CLEANUP_SCHEDULE
         #file: .env
         UPLOAD_SESSION_EXPIRED_CLEANUP_SCHEDULE="0 2 * * *"
 
+In-Progress Submission Controls
+-------------------------------
+
+IN_PROGRESS_SUBMISSION_EXPIRING_EMAIL_SCHEDULE
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    
+    *Cron schedule expression for sending reminder emails for in-progress submissions with expiring upload sessions*
+
+    .. table::
+
+        ============  =========
+        Default       Type
+        ============  =========
+        "0 2 * * *"   string
+        ============  =========
```

Comment:
Sphinx doesn't like the asterisks because it thinks they're the start of an emphasis (italic or bold). I'd suggest escaping them like this:

```suggestion
        ===============  =========
        Default          Type
        ===============  =========
        "0 \* \* \* \*"   string
        ===============  =========
```